Version 1.2.1/os/os/.programs

```diff
@@ -16,7 +16,7 @@ local function drawMenu()
    term.setCursorPos(1,1)
    term.write("Program List")
 term.setCursorPos(1,2)
-shell.run("id")
+print("#"..os.getComputerID())

```

Comment:
You are using this in an inconsistent way, above you print `Computer ID: <id>` here you use `#<id>`